README.md

```diff
@@ -25,5 +25,6 @@ Start the Bot like this:
     webcam - Get Picture of Webcam
     subscribe - Subscribe to a Sensor with an Duration
     help - Get Help
+    grammer - Get the Grammer for this Bot
```

Comment:
s/[Gg]rammer/grammar
Maybe you should use spell checking :wink: btw. nouns are written in lower case in English